src/appleseed-max-impl/appleseedsssmtl/appleseedsssmtl.cpp

```diff
@@ -828,3 +834,12 @@ HINSTANCE AppleseedSSSMtlClassDesc::HInstance()
 {
     return g_module;
 }
+
+bool AppleseedSSSMtlClassDesc::IsCompatibleWithRenderer(ClassDesc& rendererClassDesc)
+{
+    Class_ID classID = rendererClassDesc.ClassID();
```

Comment:
I had to changed it to
`return (renderer_class_desc.ClassID() == AppleseedRenderer::get_class_id()) ? true : false;`
because ClassID's operator== returns int and compiler was complaining about converting int to bool